include/zephyr/bluetooth/iso.h

```diff
@@ -140,6 +140,8 @@ extern "C" {
 /** Maximum pre-transmission offset */
 #define BT_ISO_PTO_MAX              0x0FU
 
+#define BT_ISO_VALID_BIS_BITFIELD(_bis_bitfield) ((_bis_bitfield) <= BIT_MASK(BT_ISO_BIS_INDEX_MAX))
```

Comment:
There's a `param->bis_bitfield > BIT_MASK(BT_ISO_BIS_INDEX_MAX)` check in `iso.c` that can be modified to use this macro